src/auto/tree_expander.rs

```diff
@@ -0,0 +1,178 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::object::Cast;
+use glib::object::IsA;
+use glib::signal::connect_raw;
+use glib::signal::SignalHandlerId;
+use glib::translate::*;
+use glib_sys;
+use gtk_sys;
+use std::boxed::Box as Box_;
+use std::fmt;
+use std::mem::transmute;
+use Accessible;
+use Buildable;
+use ConstraintTarget;
+use TreeListRow;
+use Widget;
+
+glib_wrapper! {
+    pub struct TreeExpander(Object<gtk_sys::GtkTreeExpander, gtk_sys::GtkTreeExpanderClass, TreeExpanderClass>) @extends Widget, @implements Accessible, Buildable, ConstraintTarget;
+
+    match fn {
+        get_type => || gtk_sys::gtk_tree_expander_get_type(),
+    }
+}
+
+impl TreeExpander {
+    pub fn new() -> TreeExpander {
+        assert_initialized_main_thread!();
+        unsafe { Widget::from_glib_none(gtk_sys::gtk_tree_expander_new()).unsafe_cast() }
+    }
+}
+
+impl Default for TreeExpander {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub const NONE_TREE_EXPANDER: Option<&TreeExpander> = None;
+
+pub trait TreeExpanderExt: 'static {
+    fn get_child(&self) -> Option<Widget>;
+
+    //fn get_item(&self) -> /*Unimplemented*/Option<Fundamental: Pointer>;
```

Comment:
Needs a `type GLib.Object` annotation and then it would be generated

Reply:
Added to the gtk's broken annotations issue https://gitlab.gnome.org/GNOME/gtk/-/issues/3281